easybuild/easyconfigs/d/DROP/DROP-1.0.3-foss-2020b-R-4.0.3.eb

```diff
@@ -23,6 +23,7 @@ dependencies = [
     ('HTSlib', '1.11'),  # for tabix
     ('SAMtools', '1.11'),
     ('BCFtools', '1.11'),
+    ('Pandoc', '2.13', '', True),
```

Comment:
I feel it might still be worth leaving it commented, if it's was a niche usecase, and the fact that our PowerPC friends can't use it still. I don't know the software well enough, so, I'll leave it up to you to decide what's best.